Basic mono repository, docker setup.

rschoonheim opened the pull request.

Commit:
Ignore inteliliJ files.

Commit:
Mono repo setup.

Commit:
Update readme.md

rschoonheim committed:
Merge pull request #1 from rschoonheim/development

Development

rschoonheim commented on services/frontend/src/main.ts:
```diff
@@ -38,3 +38,4 @@ app.component('Tag', Tag);
 const pinia = createPinia();
 app.use(pinia).mount('#app');
 
+console.log(123);
```
Remove

Commit:
Remove console.log

lucasjellema commented:
Hi Romano, Thanks for your work. I was wondering: could you please provide some additional instructions - probably best in the readme.md - on how to get the front end application running? I see you have moved around some files and you have introduced Docker (I am not quite sure yet for what purpose). A little more explanation would be helpful to me. After all, I would also like to be able to continue work on this application ;-) kind regards, Lucas

rschoonheim committed:
Update main.

rschoonheim commented:
Hi Lucas, 

Thank you for responding to this pull request. I added additional instructions on how to get the front-end up & running in the `README.md`. 

### File movement
The only certainty is uncertainty. To enhance the agilability of a project we have to embrace uncertainties. Embracing uncertainties is really hard. There is a thin line between acceptable & over engineering. KISS, YAGNI, GRASP & SOLID right? 

This isn’t the place to discuss philosophy..

This pull request aims for setting clear boundaries between different services. A service is a collection of source code that can be compiled, transpiled, interpreted independent from other services. The event calendar was moved into `services/frontend` to make it a service.

There will be shared code between services. To support this there is a directory “shared” containing source code collections. Each shared collection doesn’t have dependencies to shared or service collections. Only services can have a dependency on shared code. 

![a drawio (2)](https://github.com/lucasjellema/vue-event-agenda-prototype/assets/88203507/f4ba2233-8de7-4d3e-94bc-7ca56e63ef90)

### Introducing docker
No special reason why I introduced docker. I have multiple applications running on my local machine. Docker makes it manageable🙂.
